lib/forms/quantum_forms.js

```diff
@@ -34,7 +34,7 @@ export class AbstractResource {
 		change: (event) => {
 			this.onChange(event)
 		},
-		keyup: () => {
+		keypress: () => {
```

Comment:
You should remove this, Just add keypress.

Reply:
Done